app/jobs/deposit_job.rb

```diff
@@ -99,5 +99,11 @@ def existing_structural_for(druid)
     cocina_json = JSON.parse(cocina_str)
     cocina = Cocina::Models.build(cocina_json)
     cocina.structural.contains
+  rescue SdrClient::Find::Failed
+    # This exception is rescued when the find operation fails.
+    #
+    # This is known to happen in H2 when dealing with an object version that
+    # does not yet have a druid. When this happens, return an empty list.
+    []
```

Comment:
This may or may not be the right thing to do here. Reviews welcome!

Reply:
Can we check for the druid before trying the find?  Then we can let this exception raise so that it shows up in HB?